client/signup/config/steps.js

```diff
@@ -87,5 +88,17 @@ module.exports = {
 		},
 		dependencies: [ 'siteSlug' ],
 		providesDependencies: [ 'theme', 'images' ]
+	},
+
+	'jetpack-user': {
+		stepName: 'jetpack-user',
+		apiRequestFunction: stepActions.createAccount,
+		providesToken: true,
+		props: {
+			headerText: i18n.translate( 'Create an account for Jetpack' ),
+			subHeaderText: i18n.translate( 'You\'re moments away from connecting Jetpack.' )
+		},
+		dependencies: [ 'jetpackRedirect' ],
+		providesDependencies: [ 'bearer_token', 'username', 'jetpackRedirect' ]
```

Comment:
That's what I had initially thought too, but I simply couldn't get it to work without it.  /shrug